docs/src/prototypes/chatMessages/ThreadedMessages/theme.tsx

```diff
@@ -0,0 +1,128 @@
+import { ThemeInput, ChatItem, Input } from '@stardust-ui/react'
+import classNames from './classNames'
+
+const customizedTheme: ThemeInput = {
+  componentStyles: {
+    ChatItem: {
+      root: ({ props: p, theme: { siteVariables } }) => ({
+        [`& .${ChatItem.slotClassNames.message}`]: {
+          width: '100%',
+        },
+        [`&.${classNames.threadReplies.chatItem}`]: {
+          padding: 0,
+          backgroundColor: siteVariables.colors.grey[50],
+        },
+        [`& .${classNames.threadReplies.chatItemMessage}`]: {
+          margin: 0,
+        },
+        [`& .${classNames.threadReplies.gutter}`]: {
+          left: '15px',
+          zIndex: '1111',
+        },
+      }),
+    },
+    ChatMessage: {
+      root: ({ props: p, theme: { siteVariables } }) => ({
+        [`&.${classNames.threadedMessage.thread}`]: {
+          padding: 0,
+          width: '100%',
+        },
+        [`&.${classNames.threadedMessage.threadBody}`]: {
+          width: '100%',
+          minWidth: '100%',
+          padding: 0,
+          margin: 0,
+          borderBottom: `1px solid ${siteVariables.colors.grey[100]}`,
+          borderBottomLeftRadius: 0,
+          borderBottomRightRadius: 0,
+
+          [`& .${classNames.threadedMessage.innerContent}`]: {
+            padding: '8px 16px',
+          },
+          [`& .${classNames.threadedMessage.author}`]: {
+            padding: '5px 5px 5px 0',
+          },
+          [`& .${classNames.threadedMessage.timestamp}`]: {
+            padding: '5px',
+            color: siteVariables.colors.grey[350],
+          },
+        },
+        [`&.${classNames.threadReplies.message}`]: {
+          width: '100%',
+          minWidth: '100%',
+          margin: '1px 0',
+          paddingLeft: '60px',
+          backgroundColor: siteVariables.colors.grey[50],
```

Comment:
Can we use values from `colorScheme` instead?

Reply:
yup